bindings/src/main/java/io/pravega/storage/extendeds3/ExtendedS3StorageConfig.java

```diff
@@ -93,6 +97,7 @@ private ExtendedS3StorageConfig(TypedProperties properties) throws Configuration
         String givenPrefix = Preconditions.checkNotNull(properties.get(PREFIX), "prefix");
         this.prefix = givenPrefix.endsWith(PATH_SEPARATOR) ? givenPrefix : givenPrefix + PATH_SEPARATOR;
         this.useNoneMatch = properties.getBoolean(USENONEMATCH);
+        this.smallObjectThreshold = properties.getInt(SMALL_OBJECT_THRESHOLD);
```

Comment:
If the threshold is configurable, then please add it into config.properties as commented line with description.

Reply:
+1. In `config.properties`, please add recommended values and explain what could happen if setting it too small or too large.

Reply:
Do we use any kind of prefix to highlight which component this config property from? Something like `extendeds3.`

Reply:
yes, there are namespaces for configs. so in config.properties it's going to be something like extendeds3.smallObjectThreshold, which will be only picked up by ExtendedS3Config

Reply:
Good catch @kevinhan88 .Added to config value.